tokens.xml

```diff
@@ -2070,6 +2098,22 @@ At the beginning of your upkeep, sacrifice another creature. If you can't, this
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Dragon Token     </name>
+            <text>Flying</text>
+            <prop>
+                <colors></colors> <!--This card's color is Gold-->
```

Comment:
while I agree that adding it to the real color property is undesirable this comment feels out of place, either add it to the card text (might be actually useful) or just omit it

Reply:
Agreed. Made it a parenthetical in the text field.

Reply:
I like the new addition, technically tokens aren't cards but I think this is fine